imguiWrapper.cpp

```diff
@@ -244,6 +244,21 @@ IggBool iggSliderInt(char const *label, int *value, int minValue, int maxValue,
    return ImGui::SliderInt(label, value, minValue, maxValue, format) ? 1 : 0;
 }
 
+IggBool iggDragInt( char const *label, int *value, int speed, int min, int max, const char* format)
```

Comment:
Please have the order be "char const *format" - I'm quite picky about that ;)
See also other occurence in iggDragFloat() and header.